openmp/libomptarget/plugins-nextgen/common/PluginInterface/PluginInterface.h

```diff
@@ -1010,6 +1018,10 @@ struct GenericPluginTy {
   /// Number of devices available for the plugin.
   int32_t NumDevices = 0;
 
+  /// Offset which when added to a DeviceId will yield a unique, user-observable
+  /// device identifier.
```

Comment:
Should this be reworded in the sense that makes it clear that this is (mainly/only?) in scenarios with multiple different supported devices?

Reply:
Valid point, I'm not pushing that naming or description.

Could also be name the corresponding variable PluginDeviceStartIndex (or similar); "global" was my go-to as a globally / unique DeviceId we wanted to determine.
Or was your initial thought that a clarification in the comment would suffice?

Reply:
> Valid point, I'm not pushing that naming or description.
> 
> Could also be name the corresponding variable PluginDeviceStartIndex (or similar); "global" was my go-to as a globally / unique DeviceId we wanted to determine. Or was your initial thought that a clarification in the comment would suffice?

I like PluginDeviceStartIndex because that partially corresponds to the variable name where it originates from (libomptarget rtl.cpp).

Reply:
Since we are in the `GenericPluginTy` namespace, I'll drop the Plugin prefix and suggest: `DeviceIdStartIndex` and also go with a more verbose description -- so it should be more clear when this is actually useful.